### PR TITLE
Support YML file argument

### PR DESCRIPTION
**Pull request by comonadd**
Implements #1

### CODE REVIEW

**Comment by rajasegar** [comment]
@comonadd  can you please rebase to fix the conflicts?

**Comment by comonadd** [comment]
Fixed

File: index.js [hunk 1]
```diff
@@ -82,14 +80,33 @@ function updateTheme(data, theme, ymlPath, preview = false) {
...
       if (err) throw err;
     });
 }
 
+function updateTheme(data, theme, ymlPath, preview = false) {
+  const isSpecificFile =
+    fs.existsSync(theme) && !fs.lstatSync(theme).isDirectory();
+  const themePath = isSpecificFile
+    ? theme
+    : path.join(__dirname, `themes/${theme}.yml`);
```
Comment (JuanVqz) [inline index.js:106]:
```suggestion
    : themeFilePath(theme);
```

File: index.js [hunk 2]
```diff
@@ -10,7 +10,6 @@ const {
   alacrittyFileExists,
   alacrittyTemplatePath,
   isWindows,
-  themeFilePath,
```
Comment (JuanVqz) [inline index.js:13]: please, revert this and use it below

**Review by JuanVqz — CHANGES_REQUESTED** [review]

File: index.js (hunk 2 above)
Comment (comonadd) [reply index.js]: Done

**Review by JuanVqz — APPROVED** [review]

File: index.js (hunk 2 above)
Comment (JuanVqz) [reply index.js]: @comonadd Thank you for your PR! 🔥

**Pull request merged**